### ISSUE TITLE
Ergonomy : "all in one" groups management page

### ISSUE THREAD

**Issue by kazemar** [kalisio/kdk#448]
Creating a unique page to manage groups would save clicks and time :

**1. Option to see all members of a group**
For example by making a drop-down menu that appears when clicking on the text telling the numbers of members.
This should stay untill a new click hides it : so we are able to compare the members of all groups.
Beside of the list it will be a tool to remove members from the group.

**2. Search tool in every group abstract**
This permits to add members directly in groups, without leaving the page.

It could be like in the screenshot below : an arrow to open the drop-down list of members, and a search field to find members to add in the group.
![image](https://user-images.githubusercontent.com/6574960/129579123-b1817b22-f5de-4b97-9c9e-700c103cc195.png)

**Comment by claustres** [comment]
Yes it would be useful, there is a plan to create a new component to manage groups/tags in a similar way: list all members in or outside, select a set of members to add/remove, etc.

**Comment by cnouguier** [comment]
Won't fix in the KDK.
